Ported/dots-training-2022-03-na-group1_BucketBrigade/Assets/Scripts/Systems/BucketFillSystem.cs

```diff
@@ -0,0 +1,32 @@
+using Unity.Entities;
+
+public partial class BucketFillSystem : SystemBase
+{
+    protected override void OnUpdate()
+    {
+        var currentTime = UnityEngine.Time.realtimeSinceStartup;
```

Comment:
Time.elapsed is the version that's modeled in the tutorial.

Reply:
done